package.json

```diff
@@ -64,10 +64,7 @@
     "link-lsp": "lerna exec yarn link @salesforce/aura-language-server @salesforce/lwc-language-server @salesforce/lightning-lsp-common --scope salesforcedx-vscode-lightning && lerna exec yarn link @salesforce/lwc-language-server @salesforce/lightning-lsp-common --scope salesforcedx-vscode-lwc",
     "unlink-lsp": "lerna exec yarn unlink @salesforce/aura-language-server @salesforce/lwc-language-server @salesforce/lightning-lsp-common --scope salesforcedx-vscode-lightning && lerna exec yarn unlink @salesforce/lwc-language-server @salesforce/lightning-lsp-common --scope salesforcedx-vscode-lwc"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/forcedotcom/salesforcedx-vscode.git"
-  },
+  "repository": "forcedotcom/salesforcedx-vscode",
```

Comment:
This is the format expected by release-management/dependabot-automerge
Apparently this is a valid format https://docs.npmjs.com/cli/v8/configuring-npm/package-json#repository
> For GitHub, GitHub gist, Bitbucket, or GitLab repositories you can use the same shortcut syntax you use for npm install